src/Components/Home.js

```diff
@@ -24,13 +25,16 @@ function Home() {
       {showProgress && <ProgressBar mode="indeterminate" />}
       {list.map((user, key) => (
         <a href={`${user.username}`} key={`avatar-${key}`}>
-          <Avatar
-            image={user.avatar}
-            shape="circle"
-            size="xlarge"
-            className="p-m-2"
-            imageAlt={user.username}
-          />
+          <div className = "avatar-div" data-tip= {user.username} data-for='toolTip1' data-place='bottom'>
+            <Avatar
+              image={user.avatar}
+              shape="circle"
+              size="xlarge"
+              className="p-m-2"
+              imageAlt={user.username}
+            />
+          </div>
+          <ReactTooltip id="toolTip1" />
```

Comment:
I would use a more descriptive name here

Reply:
you mean id ? Maybe you can suggest a name

Reply:
puhhh maybe `toolTipUserName` but I'm open for other suggestions